<ui version="4.0" >
 <class>QPitch</class>
 <widget class="QMainWindow" name="QPitch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QPitch</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qpitch.qrc" >:/icons/qpitch.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLogView" native="1" name="widget_qlogview" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QGroupBox" name="groupBox_tuning" >
      <property name="title" >
       <string>Tuning</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_note" >
         <property name="text" >
          <string>Note</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="lineEdit_note" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor" >
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="maxLength" >
          <number>10</number>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_frequency" >
         <property name="text" >
          <string>Frequency</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lineEdit_frequency" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor" >
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="maxLength" >
          <number>10</number>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="QOsziView" native="1" name="widget_qosziview" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about" />
    <addaction name="action_aboutQt" />
   </widget>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;QPitch</string>
    </property>
    <addaction name="action_preferences" />
    <addaction name="separator" />
    <addaction name="action_compactView" />
    <addaction name="separator" />
    <addaction name="action_quit" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quits the application</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_aboutQt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_preferences" >
   <property name="text" >
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="action_compactView" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Compact View</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QOsziView</class>
   <extends>QWidget</extends>
   <header>qosziview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLogView</class>
   <extends>QWidget</extends>
   <header>qlogview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qpitch.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>QPitch</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>290</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
